.github/scripts/commit-repo.sh

```diff
@@ -11,6 +11,7 @@ if [ -n "$(git status --porcelain)" ]; then
     git push
 
     curl https://purge.jsdelivr.net/gh/keiyoushi/extensions@repo/index.min.json
+    curl https://purge.jsdelivr.net/gh/keiyoushi/extensions@repo/index.html
```

Comment:
```suggestion
```
since this file isn't something that will be used by many people, no need to force clear jsdelivr cache